lib/test/requirements_resolver.rb

```diff
@@ -33,22 +35,35 @@ def dependency_map
         Test::Tasks::RunVitest => Test::Tasks::PnpmInstall,
         Test::Tasks::RunAnnotate => Test::Tasks::SetupDb,
         Test::Tasks::RunTypelizer => Test::Tasks::BuildFixtures,
-        Test::Tasks::ConvertSchemasToTs => nil,
+        Test::Tasks::ConvertSchemasToTs => Test::Tasks::SetupDb,
         Test::Tasks::RunBrakeman => nil,
         Test::Tasks::RunDatabaseConsistency => Test::Tasks::SetupDb,
         Test::Tasks::RunImmigrant => Test::Tasks::SetupDb,
         Test::Tasks::RunRubocop => nil,
         Test::Tasks::RunUnitTests => Test::Tasks::CreateDbCopies,
         Test::Tasks::RunApiControllerTests => Test::Tasks::CreateDbCopies,
         Test::Tasks::RunFileSizeChecks => Test::Tasks::CompileJavaScript,
-        Test::Tasks::RunFeatureTests => [
+        Test::Tasks::RunFeatureTestsA => [
+          Test::Tasks::DivideFeatureSpecs,
           Test::Tasks::CreateDbCopies,
           Test::Tasks::CompileJavaScript,
+          Test::Tasks::StartPercy,
+        ],
+        Test::Tasks::RunFeatureTestsB => [
+          Test::Tasks::DivideFeatureSpecs,
+          Test::Tasks::CreateDbCopies,
+          Test::Tasks::CompileJavaScript,
+          Test::Tasks::StartPercy,
+          Test::Tasks::RunApiControllerTests,
```

Comment:
`RunFeatureTestsB` uses the `_api` database, so we have to wait until those tests are complete, so that there aren't conflicts over the DB with the API controller tests.